apps/docs/package.json

```diff
@@ -8,6 +8,7 @@
     "start": "next start",
     "lint": "next lint",
     "build:sitemap": "node ./internals/generate-sitemap.mjs",
+    "build:embeddings": "tsx scripts/generate-embeddings.ts",
```

Comment:
Note [`tsx`](https://github.com/esbuild-kit/tsx) was required here (as opposed to `ts-node`) in order to properly deal with ESM+CommonJS dependencies used together.